opcua/common/node.py

```diff
@@ -525,30 +525,30 @@ def delete(self, delete_references=True):
 
     def add_folder(self, nodeid, bname):
         return  opcua.common.manage_nodes.create_folder(self, nodeid, bname)
- 
+
     def add_object(self, nodeid, bname, objecttype=None):
         return opcua.common.manage_nodes.create_object(self, nodeid, bname, objecttype)
- 
+
     def add_variable(self, nodeid, bname, val, varianttype=None, datatype=None):
         return opcua.common.manage_nodes.create_variable(self, nodeid, bname, val, varianttype, datatype)
- 
+
     def add_object_type(self, nodeid, bname):
         return opcua.common.manage_nodes.create_object_type(self, nodeid, bname)
- 
+
     def add_variable_type(self, nodeid, bname, datatype):
         return opcua.common.manage_nodes.create_variable_type(self, nodeid, bname, datatype)
- 
+
     def add_data_type(self, nodeid, bname, description=None):
         return opcua.common.manage_nodes.create_data_type(self, nodeid, bname, description=None)
- 
+
     def add_property(self, nodeid, bname, val, varianttype=None, datatype=None):
         return opcua.common.manage_nodes.create_property(self, nodeid, bname, val, varianttype, datatype)
- 
+
     def add_method(self, *args):
         return opcua.common.manage_nodes.create_method(self, *args)
- 
+
     def add_reference_type(self, parent, nodeid, bname):
         return opcua.common.manage_nodes.create_reference_type(parent, nodeid, bname)
-    
-    def call_method(parent, methodid, *args):
+
+    def call_method(self, parent, methodid, *args):
```

Comment:
This was not an error. Self was called parent her. I admit this was confusing. But you need to remind listener arg if you add self. Currently your pr breaks tests